magento-research/pwa-studio
# [Doc] Venia UPWARD spec file explainer

## Description

Add verbose comments to Venia's UPWARD spec file to help explain what each piece does.

## Related Issue

Closes #1130 

## Verification Steps

1. Open Venia's UPWARD spec file
2. See comments

## How Have YOU Tested this?

n/a

## Screenshots / Screen Captures (if appropriate):

n/a

## Checklist:
